### PR TITLE
docs: refine some desc

### PR DESCRIPTION
**Pull request by zmstone**
Found some descriptions are not perfect when trying to translate to zh in emqx-i18n.git
see also: https://github.com/emqx/emqx-i18n/pull/1

### CODE REVIEW

File: rel/i18n/emqx_ft_schema.hocon [hunk 1]
```diff
@@ -7,16 +7,16 @@ This toggle also affects the availability of the File Transfer REST API and
...
 init_timeout.desc:
-"""Timeout for initializing the file transfer.<br/>
-After reaching the timeout, `init` message will be acked with an error"""
+"""Timeout for EMQX to initialize the file transfer.<br/>
+After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `init` will containe error code (0x80)."""
 
 assemble_timeout.desc:
 """Timeout for assembling and exporting file segments into a final file.<br/>
-After reaching the timeout, `fin` message will be acked with an error"""
+After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `fin` will containe error code (0x80)"""
```
Comment (savonarola) [inline rel/i18n/emqx_ft_schema.hocon:15]:
```suggestion
After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `fin` will contain error code (0x80)"""
```

File: rel/i18n/emqx_ft_schema.hocon [hunk 2]
```diff
@@ -7,16 +7,16 @@ This toggle also affects the availability of the File Transfer REST API and
 storage-dependent background activities (e.g. garbage collection)."""
 
 init_timeout.desc:
-"""Timeout for initializing the file transfer.<br/>
-After reaching the timeout, `init` message will be acked with an error"""
+"""Timeout for EMQX to initialize the file transfer.<br/>
+After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `init` will containe error code (0x80)."""
```
Comment (savonarola) [inline rel/i18n/emqx_ft_schema.hocon:11]:
```suggestion
After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `init` will contain error code (0x80)."""
```

File: rel/i18n/emqx_ft_schema.hocon [hunk 3]
```diff
@@ -7,16 +7,16 @@ This toggle also affects the availability of the File Transfer REST API and
...
 
 assemble_timeout.desc:
 """Timeout for assembling and exporting file segments into a final file.<br/>
-After reaching the timeout, `fin` message will be acked with an error"""
+After reaching the timeout (e.g. due to system is overloaded), the PUBACK message for `fin` will containe error code (0x80)"""
 
 store_segment_timeout.desc:
 """Timeout for storing a file segment.<br/>
-After reaching the timeout, message with the segment will be acked with an error"""
+After reaching the timeout (e.g. due to system overloaded), the PUBACK message for the PUBLISH will containe error code (0x80)."""
```
Comment (savonarola) [inline rel/i18n/emqx_ft_schema.hocon:19]:
```suggestion
After reaching the timeout (e.g. due to system overloaded), the PUBACK message for the PUBLISH will contain error code (0x80)."""
```

File: rel/i18n/emqx_mgmt_api_nodes.hocon [hunk 4]
```diff
@@ -11,12 +11,12 @@ get_node_info.label:
 """Get node info"""
 
 get_node_metrics.desc:
-"""Get node metrics"""
+"""Get node run-time counter metrics. Such as received or sent bytes or messages, the number of succeeded or failed authentication or authorizations, etc."""
```
Comment (savonarola) [inline rel/i18n/emqx_mgmt_api_nodes.hocon:14]:
```suggestion
"""Get node run-time counter metrics. Such as received or sent bytes or messages, the number of succeeded or failed authentications or authorizations, etc."""
```

File: rel/i18n/emqx_mgmt_api_trace.hocon [hunk 5]
```diff
@@ -50,35 +50,35 @@ file_size.desc:
 file_size.label:
 """file size"""
 
-modification_date.desc:
-"""the modification and last access times of a file"""
-modification_date.label:
-"""last access time"""
+file_mtime.desc:
+"""The time the last time this file is modified."""
```
Comment (savonarola) [inline rel/i18n/emqx_mgmt_api_trace.hocon:54]:
```suggestion
"""The last time this file is modified."""
```

File: rel/i18n/emqx_bridge_iotdb.hocon [hunk 6]
```diff
@@ -42,7 +42,9 @@ config_is_aligned.label:
 """Align Timeseries"""
 
 config_device_id.desc:
-"""A fixed device name this data should be inserted for. If empty it must either be set in the rule action, the message itself, or it will be extracted from the topic."""
+"""The IoTDB device ID this data should be inserted for.
+If left empty, the MQTT message payload must containe a `device_id` field,
```
Comment (keynslug) [inline rel/i18n/emqx_bridge_iotdb.hocon:46]:
```suggestion
If left empty, the MQTT message payload must contain a `device_id` field,
```

File: rel/i18n/emqx_bridge_iotdb.hocon [hunk 7]
```diff
@@ -42,7 +42,9 @@ config_is_aligned.label:
...
+or EMQX's rule-engine SQL must output a `device_id` field."""
```
Comment (keynslug) [inline rel/i18n/emqx_bridge_iotdb.hocon:47]: Maybe `produce` would work better?

**Review by savonarola — APPROVED** [review]

**Pull request merged**